doc/user/installation.rst

```diff
@@ -182,6 +182,14 @@ and the configuration files in :file:`/usr/local/etc`. The :file:`/usr/local/`
 installation prefix and other directories may be changed using the following
 options to the configuration script.
 
+.. option:: --enable-multipath=X
+
+Compile FRR with up to X way ECMP supported.  This number can be from 0-999.
```

Comment:
block should be indented 3 spaces

Reply:
@donaldsharp  want to address @qlyoung ? he did "block" this PR! ;)

Reply:
roger mc dodger